events/class_tests/create_events.py

```diff
@@ -77,3 +78,23 @@ def test_user_not_exists_when_saving_form(self, valid_event_data, user0):
                 form.save()
             else:
                 assert False
+
+    @pytest.fixture
+    def signed_up_user_details(self):
+        return {'email': 'testUser1@mta.ac.il', 'password': 'PasswordU$er123'}
+
+    @pytest.fixture
+    def sign_in(self, client, signed_up_user_details):
+        return client.post('/login/', data=signed_up_user_details)
+
+    def test_valid_delete_event(self, client, sign_in):
+        event_id = 1
+        response = client.get(f"/event/delete_event/{event_id}")
+        assert response.status_code == 200
+        assert loads(response.content)["result"] == "success"
+
+    def test_invalid_delete_event(self, client, sign_in):
+        event_id = 2
+        response = client.get(f"/event/delete_event/{event_id}")
+        assert response.status_code == 200
+        assert loads(response.content)["result"] == "fail"
```

Comment:
These test functions are pretty much the same. Maybe you can write them as one test called `test_delete_event` using parameterize with the parameters- `event_id` and `result` and two ids such as "valid_delete_event" and "invalid_delete_event"

Reply:
Great 😄 
Done.